content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1476,21 +2208,38 @@ Defines an index in the database.
 
 #### Remarks
 
+##### Relational databases
+
 - Corresponding database type: `INDEX`
+- It is currently not possible to provide more configuration options to the index:
+  - PostgreSQL
+    - Define index fields as expressions (e.g. `CREATE INDEX title ON public."Post"((lower(title)) text_ops);`)
+    - Specify index methods with `USING`; PostgreSQL supports these index methods: B-tree, hash, GiST, and GIN; Prisma uses B-Tree by default
+    - Define partial indexes with `WHERE`
+    - Create indexes concurrently with `CONCURRENTLY`
+  - MySQL
+    - Specify index methods with `USING`; MySQL supports these index methods: B-tree, hash; Prisma uses B-Tree by default
```

Comment:
Unrelated to PR: All these should have issues that can be linked here.

Reply:
https://github.com/prisma/docs/issues/2021